### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by frznlogic**
Added flush method to Serial. 
Added Linux elseif to not redefine true/false

### CODE REVIEW

File: src/serial.cc [hunk 1]
```diff
@@ -88,3 +88,7 @@ size_t Serial_::println(void) {
 uint8_t Serial_::begin(uint16_t port) {
   return gSerialMock->begin(port);
 }
+
+void Serial_::flush() {
+    return;
```
Comment (aYasukiIkeuchi) [inline src/serial.cc]: * Please call `gSerialMock->flush()`
* Please use two spaces for indent. (Sorry, it's just a trivial comment.)

File: include/arduino-mock/arduino.h [hunk 2]
```diff
@@ -17,7 +17,9 @@ extern "C"{
 #define OUTPUT 0x1
 #define INPUT_PULLUP 0x2
 
-#ifndef WIN32
+#ifdef WIN32
+#elif linux
```
Comment (aYasukiIkeuchi) [inline include/arduino-mock/arduino.h]: :+1:

**Comment by aYasukiIkeuchi** [comment]
- [ ] Could you add a test to 'test/serial_unittest.cc'?

File: include/arduino-mock/arduino.h (hunk 2 above)
Comment (ikeyasu) [inline include/arduino-mock/arduino.h]: :+1:

File: src/serial.cc (hunk 1 above)
Comment (ikeyasu) [inline src/serial.cc]: - [ ] Please call gSerialMock->flush()
- [ ] Please use two spaces for indent. (Sorry, it's just a trivial comment.)

**Comment by ikeyasu** [comment]
- [ ] Could you add a test to 'test/serial_unittest.cc'?

NOTE: I used my another account, aYasukiIkeuchi. I removed and re-create comments. I'm sorry bothering you.

**Comment by frznlogic** [comment]
Thanks for your quick comments! I've taken care of them, the test is in test/serial_unittest.cc. 

On the same note, the test case names and test names seem a bit off. Should I fix those as well?

**Comment by ikeyasu** [comment]
Thank you for your modifications. LGTM. I will merge your pull-request.

> On the same note, the test case names and test names seem a bit off. Should I fix those as well?

It's fine for me. Thanks.

**Comment by frznlogic** [comment]
Thanks. I am working on some more additions, will send a new pull request later when I get to it.

**Pull request merged**